docker/hugo.dockerfile

```diff
@@ -14,17 +18,26 @@ RUN wget --quiet "https://github.com/gohugoio/hugo/releases/download/v${HUGO_VER
 WORKDIR /src
 COPY ./hugo/ /src
 
+# GH actions triggered runs will overwrite HUGO_ENV_ARG 
+ARG HUGO_ENV_ARG="Local dev build"
+
+# Set the combined environment variable for hugo build
+ARG ENV_INFO="${HUGO_ENV_ARG}|${JBROWSE_VERSION}"
+
 RUN mkdir /target && \
-    hugo -d /target
+    hugo -d /target --minify -e "${ENV_INFO}"
```

Comment:
I am a bit worried about using the `-e` option, since it may be relied upon by `hugo`. For the selection of the configuration file for example, or the [hugo.IsDevelopment](https://gohugo.io/functions/hugo/isdevelopment/) function. Do you think it would be possible to pass regular environment variables to the build and then use [os.Getenv](https://gohugo.io/functions/os/getenv/) This would allow you to pass as many as you please, an ease the parsing logic in `version_info.html`

Reply:
Great idea, I copied the approach by the pathogens portal with using -e, but I really like this idea. As you say, it will simplify the logic in `version_info.html `a lot. 
I will work on making this change and let you know when it is done :)